scripts/grab-data-from-central.sh

```diff
@@ -40,13 +40,16 @@ main() {
 
     roxctl -e "${api_endpoint}" -p "${ROX_PASSWORD}" central backup --output "${dest}"
 
-    if ! [ -x "$(command -v rocksdbdump)" ]; then
-        go install ./tools/rocksdbdump
-    fi
+    # With Postgres we no longer take RocksDB dumps
+    if [ -z "${ROX_POSTGRES_DATASTORE}" ] || [ "${ROX_POSTGRES_DATASTORE}" = false ]; then
+      if ! [ -x "$(command -v rocksdbdump)" ]; then
```

Comment:
We still have rocksdb even with Postgres DB enabled. For the purpose of debugging, we'd better still collect the remaining rocksdb dump.